src/Time/Bool.hs

```diff
@@ -1,39 +1,18 @@
-{-# LANGUAGE AllowAmbiguousTypes        #-}
-{-# LANGUAGE ConstraintKinds            #-}
-{-# LANGUAGE CPP                        #-}
-{-# LANGUAGE DataKinds                  #-}
-{-# LANGUAGE DeriveGeneric              #-}
-{-# LANGUAGE ExplicitForAll             #-}
-{-# LANGUAGE FlexibleContexts           #-}
-{-# LANGUAGE InstanceSigs               #-}
-{-# LANGUAGE GeneralizedNewtypeDeriving #-}
-{-# LANGUAGE ScopedTypeVariables        #-}
-{-# LANGUAGE TypeApplications           #-}
-{-# LANGUAGE TypeFamilies               #-}
 {-# LANGUAGE TypeInType                 #-}
 {-# LANGUAGE TypeOperators              #-}
 {-# LANGUAGE UndecidableInstances       #-}
 
 module Time.Bool
-       ( And
-       , If
-       , type (==%)
+       ( If
+       , type (&&)
        ) where
 
-import Data.Type.Equality (type (==))
-
-import Time.Rational (type (:%), Rat)
-
 -- | Type-level 'if'.
 type family If (x :: Bool) a b where
```

Comment:
Sorry, I probably need to go to doctor because I try to avoid `TypeInType` extension at any cost... I think you can replace `TypeInType` with `PolyKinds` and write if as `If (x :: Bool) (a :: k) (b :: k) :: k`. I guess this should work...

Reply:
After discussion with @int-index I changed my mind. So you can keep `-XTypeInType`. Well, probably no harm in it.